examples/buttons/index.html

```diff
@@ -19,10 +19,10 @@ <h1 class="heading-1">ClientKit Demos</h1>
     <section class="buttons">
       <h1>Buttons</h1>
       <h2 class="heading-3">Default Buttons</h2>
-      <button class="btn btn-primary" type="button">Primary</button>
+      <button class="button" type="button">Primary</button>
       <button class="btn btn-secondary" type="button">Secondary</button>
       <h2 class="heading-3">Outline Buttons</h2>
-      <button class="btn btn-primary-outline" type="button">Primary</button>
+      <button class="button-outline" type="button">Primary</button>
```

Comment:
My only concern with having no base style for a button is that you're duplicating a lot of the common code that exists for buttons.